libr/bin/p/bin_dex.c

```diff
@@ -1372,7 +1385,7 @@ static void parse_dex_class_method(RBinFile *bf, RBinDexClass *c, RBinClass *cls
 									s,
 									handler_addr);
 							}
-							free (s);
+							s = NULL; // free (s);
```

Comment:
memory leak?

Reply:
nope, because its not freed until the the rbindex obj is freed